packages/common/src/components/WideSandbox/elements.ts

```diff
@@ -84,7 +95,7 @@ export const SandboxInfo = styled.div`
   left: -1px;
   right: -1px;
   padding: 0.75rem;
-  height: 130px;
```

Comment:
There is some visual glitchiness if this one is removed in the explore page. Maybe we can put it back to `130px`? Not sure what breaks for the profile page in that case.

Reply:
It deosn't break. It just has more empty space if there is no description

Added back